firebase_cloud_messenger.gemspec

```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |spec|
   spec.email         = ["vdevendra@patientslikeme.com"]
   spec.homepage      = "https://github.com/patientslikeme/firebase_cloud_messenger"
 
-  spec.summary       = %q{A ruby api wrapper for the FirebaseCloudMesenger API}
+  spec.summary       = %q{A ruby api wrapper for the FirebaseCloudMessenger API}
```

Comment:
Yeesh, this is sort of gross and lazy all around. It should probably be: "A wrapper for the Firebase Cloud Messaging API"

Reply:
Agreed; indeed, it is actually `Firebase Cloud Messaging` not Messenger.